services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeController.kt

```diff
@@ -93,7 +146,27 @@ class HankeController(
     }
 
     /** Add one hanke. This method will be called when we do not have id for hanke yet */
-    @PostMapping
+    @PostMapping(consumes = [APPLICATION_JSON_VALUE], produces = [APPLICATION_JSON_VALUE])
+    @Operation(
+        summary = "Create new hanke",
+        description =
+            "Create a new hanke. A valid new hanke must comply with the restrictions defined in HankeValidator."
```

Comment:
"restrictions defined in HankeValidator" is not very helpful for someone reading this documentation.

We write documentation so we don't have to check the code and for people who can't read the code. Referring to the code is counter-productive in that sense.

I'm not sure what's the best way to document the validation requirements. Maybe we could have `@Schema` annotations on the data class, with descriptions of what's validated.

Reply:
I guess the correct way is to use Schema annotations. I'll do the fix but it will take some time.

Reply:
Schema annotations are now added in Hanke. Changed the endpoint description accordingly.